src/modules/Routing.rb

```diff
@@ -205,6 +205,35 @@ def WriteIPForwarding
       write_ipv6_forwarding(@Forward_v6)
     end
 
+    # Converts routes config as read from system into well-defined format
```

Comment:
UGH! Why do we introduce both `normalize_routes(routes)` and `convert_route_conf(route)`?